RiotSwiftUI/Modules/Room/Composer/View/Composer.swift

```diff
@@ -33,20 +31,25 @@ struct Composer: View {
     @Environment(\.theme) private var theme: ThemeSwiftUI
     
     @State private var isActionButtonShowing = false
+    @State private var isToggleButtonHidden = false
+    
+    private var isLandscapeIphone: Bool {
```

Comment:
I fear Apple is gonna reject our application because of this capital 'I' :trollface: